### ISSUE TITLE
Internal error. Unknown type in count var in some_resource.myname: *config.LocalVariable

### ISSUE THREAD

**Issue by Rotwang** [hashicorp/terraform#16014]
Hello,

it seems that local values cannot be used in the count parameter:

```
$ ls
main.tf
$ cat main.tf 
provider "template" {
  version = "~> 0.1"
}

locals {
  qux = "3"
}

data "template_file" "foo" {
  count = "${local.qux}"
  template = "${count.index}"
}

output "bar" {
  value = "${data.template_file.foo.*.rendered}"
}
$ terraform init && terraform apply
Error getting plugins: module root: 1 error(s) occurred:

* Internal error. Unknown type in count var in data.template_file.foo: *config.LocalVariable
$
```

**Comment by Rotwang** [comment]
Terraform version 0.10.3

**Comment by Rotwang** [comment]
Solved in #15987
